.github/workflows/docker-build.yaml

```diff
@@ -465,6 +477,73 @@ jobs:
             -   name: Cypress container logs
                 if: ${{ failure() }}
                 run: docker compose logs cypress
+    tests-storefront-base:
+        name: Regenerate Cypress snapshots
+        needs: [variables, build-php-fpm-image, build-storefront-image, build-elasticsearch-image]
+        if: |
+            always() && !failure() && !cancelled() &&
+            (
+                github.event.pull_request.head.repo.full_name == 'shopsys/shopsys' ||
+                github.ref_protected == true
+            ) &&
+            (needs.build-php-fpm-image.result == 'success' || needs.build-php-fpm-image.result == 'skipped') &&
+            (needs.build-storefront-image.result == 'success' || needs.build-storefront-image.result == 'skipped') &&
+            (needs.build-elasticsearch-image.result == 'success' || needs.build-elasticsearch-image.result == 'skipped')
+        runs-on: ubuntu-22.04
+        env:
+            GH_TOKEN: ${{ github.token }}
+        steps:
+            -   name: Check pull request labels
+                run: |
+                    PULL_REQUEST_LABELS=`gh api -H "Accept: application/vnd.github+json" /repos/shopsys/shopsys/issues/${{ github.event.pull_request.number }}/labels | jq '[.[].name]'`
+                    if ! echo "$PULL_REQUEST_LABELS" | jq -e '. | index("regenerate screenshots")' >/dev/null; then
+                        exit 1;
+                    fi
+            -   name: GIT checkout branch - ${{ github.head_ref || github.ref_name }}
+                uses: actions/checkout@v4
+                with:
+                    ref: ${{ github.head_ref || github.ref_name }}
+            -   name: Print github head ref
+                run: |
+                    echo ${{ github.head_ref }}
+            -   name: Print github ref name
+                run: |
+                    echo ${{ github.ref_name }}
+            -   name: Print github event.pull_request.head.ref
+                run: |
+                    echo ${{ github.event.pull_request.head.ref }}
```

Comment:
These three print steps are not necessary, I believe.

Reply:
Fixed